app/Models/User.php

```diff
@@ -86,4 +86,32 @@ public function follows()
     {
         return $this->hasMany(Follow::class);
     }
+
+    protected static function boot()
+    {
+        parent::boot();
+        static::deleting(function ($user) {
+            $user->image()->delete();
+
+            foreach ($user->comments()->get() as $comment) {
+                $comment->delete();
+            }
+
+            foreach ($user->reviews()->get() as $review) {
+                $review->delete();
+            }
+
+            foreach ($user->likes()->get() as $like) {
+                $like->delete();
+            }
+
+            foreach ($user->bookRequests()->get() as $book_request) {
+                $book_request->delete();
```

Comment:
```suggestion
                $bookRequest->delete();
```
chú ý cách đặt tên biến